src/h5web/providers/Provider.tsx

```diff
@@ -56,10 +56,12 @@ function Provider(props: Props) {
     });
   }, [api]);
 
+  const filepathMembers = api.filepath.split('/');
```

Comment:
Any risk of seeing a trailing slash?

Reply:
Sounds very unlikely: the last member is the filename and I never seen things like `file.h5/`

Reply:
I'm thinking more of HSDS, where the buckets aren't really files, but yeah, very unlikely.

Reply:
Good point ! I tried and the last member of the bucket name cannot have a trailing slash whether when loading the data in HSDS or when trying to fetch it in h5web

Reply:
Awesome 💯